src/common/types/ku_string.cpp

```diff
@@ -35,6 +35,18 @@ std::string ku_string_t::getAsString() const {
     }
 }
 
+char* ku_string_t::getAsShortCString() const {
+    return (char*)prefix;
+}
+
+char* ku_string_t::getAsCString() const {
```

Comment:
Why do you need to return as raw c string? If you want to access the data in the ku_string_t, you can use getData() API.
It is unsafe to return a char*, since we don't put a '\0' at the end of kustring.